docs/src/Header.tsx

```diff
@@ -9,12 +8,12 @@ export function Header() {
       </NavLink>
       <ul className="navbar-nav mr-auto">
         <li className="nav-item ">
-          <NavLink className="nav-link" to="/" exact>
+          <NavLink className="nav-link" to="/" >
             Home
           </NavLink>
         </li>
         <li className="nav-item  ">
-          <NavLink className="nav-link" to="/docs" activeClassName="active">
+          <NavLink className={({isActive}) => `${isActive ? "active ":"" }nav-link`} to="/docs" >
```

Comment:
These spaces are hard to catch and easy to break, it's ok for the className string to contain redundant spaces, prefer this:
```suggestion
          <NavLink className={({isActive}) => `${isActive ? "active": ""} nav-link`} to="/docs" >
```

Reply:
thank you! that is more appropriate